### PR TITLE
Add cudf.pandas demos for colab and brev

### PR DESCRIPTION
**Pull request by taureandyernv**
- [x] intro to cudf pandas demo
- [x] FSI demo
- [ ] 3rd demo

### CODE REVIEW

**Comment by beckernick** [comment]
Left a review in ReviewNB. Generally looks good -- recommend a few small tweaks before merging

**Comment by beckernick** [comment]
Initially left the review as a ReviewNB "Discussion" by accident. Migrated it to an actual review.

File: getting_started_tutorials/cudf_pandas_stocks_demo.ipynb [hunk 1]
```diff
@@ -0,0 +1,7061 @@
+{
```
Comment (taureandyernv) [inline getting_started_tutorials/cudf_pandas_stocks_demo.ipynb:1]: <ul><li>need to update this link to <a href="https://storage.googleapis.com/rapidsai/colab-data/usa_stocks_30m.parquet" target="_blank">https://storage.googleapis.com/rapidsai/colab-data/usa_stocks_30m.parquet</a></li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai-community/showcase/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (taureandyernv) [inline getting_started_tutorials/cudf_pandas_stocks_demo.ipynb:1]: <p>We need an intro here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai-community/showcase/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (taureandyernv) [inline getting_started_tutorials/cudf_pandas_stocks_demo.ipynb:1]: <p>@beckernick, do we want to say anything special about the chart?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai-community/showcase/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

[the remark this answers is not in the document]
Comment (taureandyernv) [reply getting_started_tutorials/cudf_pandas_stocks_demo.ipynb:1]: okay, i'll redo and remove

File: getting_started_tutorials/cudf_pandas_stocks_demo.ipynb [hunk 2]
```diff
@@ -0,0 +1,6878 @@
...
```
Comment (beckernick) [inline getting_started_tutorials/cudf_pandas_stocks_demo.ipynb:1]: <p>Sentence fragment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai-community/showcase/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (beckernick) [inline getting_started_tutorials/cudf_pandas_stocks_demo.ipynb:1]: <p>Typo at " on <code>max</code>. values"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai-community/showcase/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

**Pull request merged**